webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
```

Comment:
- pls enable _devtools_ for developmnet mode, and disable for prod

- also add hot reload for dev env